firmware/thruna/BMS/Test/Src/Test_StateMachine.cpp

```diff
@@ -874,19 +897,19 @@ TEST_F(BmsStateMachineTest, check_precharge_state_transitions_and_air_plus_statu
         if (test_params[i].expect_precharge_starts)
         {
             // Precharge should start
-            LetTimePass(state_machine, 10);
+            LetTimePass(state_machine, 210);
```

Comment:
210 seems to make sense, see my comments about 250 above ^
```suggestion
            LetTimePass(state_machine, 210U);
```